community/modules/compute/schedmd-slurm-gcp-v5-partition/variables.tf

```diff
@@ -17,14 +17,15 @@
 # Most variables have been sourced and modified from the SchedMD/slurm-gcp
 # github repository: https://github.com/SchedMD/slurm-gcp/tree/v5.0.3
 
-variable "slurm_cluster_name" {
+variable "deployment_name" {
+  description = "Name of the deployment."
   type        = string
-  description = "Cluster name, used for resource naming and slurm accounting."
+}
 
-  validation {
-    condition     = can(regex("(^[a-z][a-z0-9]*$)", var.slurm_cluster_name))
-    error_message = "Variable 'slurm_cluster_name' must be composed of only alphanumeric values and begin with a leter. regex: '(^[a-z][a-z0-9]*$)'."
-  }
+variable "slurm_cluster_name" {
+  type        = string
+  description = "Cluster name, used for resource naming and slurm accounting.If not provided it will default to the first 8 characters of the deployment name (removing any invalid characters)."
```

Comment:
Space is needed after the period: accounting.If

Reply:
fixed, thanks